frontend/institution/management_members.html

```diff
@@ -250,6 +250,11 @@ <h3>{{ request.sender_name}}</h3>
               <h4>Cargo atual: {{ request.office}}</h4>
               <p>Email institucional: {{ request.institutional_email}}</p>
             </div>
+            <div layout="row" layout-align="end center" flex hide show-gt-xs>
+              <p class="status-invite-adm" style="line-height: 21px; padding: 5px;" md-colors="{color: 'teal-500'}">
```

Comment:
It is better to create a new ccs class for this button with the status-invite-adm properties and the props defined at the style tag